rest-server/src/main/resources/static/index.html

```diff
@@ -167,12 +167,16 @@
                 appServiceRuntimes: [
                     {id: 'DOCKER', name: 'Docker with a Dockerfile'},
                     {id: 'DOCKER_SPRING', name: 'Docker with Spring Boot'},
+                    {id: 'DOCKER_MICRONAUT_GRADLE', name: 'Docker with Micronaut and Gradle'},
```

Comment:
Due to Brand guidelines, replace :

`Docker with Micronaut and Gradle`

with: 

`Docker with the Micronaut Framework and Gradle`